tests/strip-indentation.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const assert = require('assert');
+const expect = require('chai').expect;
+const HTMLBarsMinifier = require('../lib/htmlbars-minifier');
+
+const minifier = new HTMLBarsMinifier('foo', {
+	stripIndentation: true
+});
+
+describe('stripIndentation', () => {
+	describe('from a text node', () => {
+		it('should remove consecutive spaces after a word', () => {
+			const input = '    foo';
+
+			assert.equal(
+				minifier.processString(input),
+				'foo'
+			);
+		});
+
+		it('should remove consecutive tabs after a word', () => {
+			const input = '			foo';
+
+			assert.equal(
+				minifier.processString(input),
+				'foo'
+			);
+		});
+
+		it('should remove consecutive spaces and tabs after a word', () => {
+			const input = '				foo';
+
+			assert.equal(
+				minifier.processString(input),
+				'foo'
+			);
+		});
+	});
+
+	describe('from a DOM element', () => {
+		it('should remove consecutive spaces after a word', () => {
```

Comment:
`word` -> `DOM element`, additionally the previous comment about before & after applies here too